string ID
geomtwo/Vector bear_msrm
float64 control_signal
float64 time_access
---
float64 beta
geomtwo/Vector bear_msrm_neigh
float64 control_signal_neigh
float64 t_neigh
#Container respMsg